[WINDOWS] golem-cloud-client compile error

This came up after rebasing the current windows ci work branch to latest main 

https://github.com/laz-001/golem/actions/runs/15302145387/job/43045338446

```sh
   Compiling golem-cloud-client v0.0.0 (D:\a\golem\golem\golem-cloud-client)
error[E0432]: unresolved import `nix::sys`
  --> C:\Users\runneradmin\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\backtrace-on-stack-overflow-0.3.0\src\lib.rs:61:10
   |
61 | use nix::sys::signal;
   |          ^^^ could not find `sys` in `nix`

error[E0422]: cannot find struct, variant or union type `stack_t` in crate `libc`
  --> C:\Users\runneradmin\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\backtrace-on-stack-overflow-0.3.0\src\lib.rs:74:27
   |
74 |         let stack = libc::stack_t {
   |                           ^^^^^^^ not found in `libc`

error[E0422]: cannot find struct, variant or union type `stack_t` in crate `libc`
  --> C:\Users\runneradmin\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\backtrace-on-stack-overflow-0.3.0\src\lib.rs:79:29
   |
79 |         let mut old = libc::stack_t { ss_sp: std::ptr::null_mut(), ss_flags: 0, ss_size: 0 };
   |                             ^^^^^^^ not found in `libc`

error[E0425]: cannot find function `sigaltstack` in crate `libc`
  --> C:\Users\runneradmin\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\backtrace-on-stack-overflow-0.3.0\src\lib.rs:80:25
   |
80 |         let ret = libc::sigaltstack(&stack, &mut old);
   |                         ^^^^^^^^^^^ not found in `libc`

Some errors have detailed explanations: E0422, E0425, E0432.
For more information about an error, try `rustc --explain E0422`.
error: could not compile `backtrace-on-stack-overflow` (lib) due to 4 previous errors
warning: build failed, waiting for other jobs to finish...
Error while executing command, exit code: 101
Error: Process completed with exit code 105.
```

--- a/.gitignore
+++ b/.gitignore
@@ -16,9 +16,8 @@ test-components/csharp-1/bin/
 test-components/csharp-1/obj/
 test-components/python-1/__pycache__/
 test-components/tinygo-wasi*/tinygo_wasi/
-golem-client/*
-!golem-client/Cargo.toml
-!golem-client/build.rs
+golem-client/openapi/*
+golem-cloud-client/openapi/*
 logs
 data
 .DS_Store

--- a/cloud-service/Cargo.toml
+++ b/cloud-service/Cargo.toml
@@ -88,4 +88,4 @@ futures = { workspace = true }
 testcontainers = { workspace = true }
 testcontainers-modules = { workspace = true }
 test-r = { workspace = true }
-backtrace-on-stack-overflow = "0.3.0"
+

--- a/cloud-service/tests/it_tests.rs
+++ b/cloud-service/tests/it_tests.rs
@@ -244,7 +244,6 @@ async fn create_project_grant(
 
 // TODO: split these into separate, isolated tests
 async fn test_services(config: &CloudServiceConfig) {
-    unsafe { backtrace_on_stack_overflow::enable() };
     let services: Services = Services::new(config).await.unwrap();
 
     // check that default plan gets created

